File: LEVEL1/완주하지 못한 선수.java
import java.io.*;
import java.util.*;

class Solution {
    public String solution(String[] participant, String[] completion) {
        String answer = "";
        String s = "";
        
        Arrays.sort(participant);
        Arrays.sort(completion);
        
        int i = 0;
        
        while(i < completion.length) {
            if(!completion[i].equals(participant[i])) {
                s = participant[i];
                break;
            }
            
            else {
                i++;
            }
        }
        
        if(!s.equals("")) {
            answer = s;
        }
        
        else {
            answer = participant[participant.length - 1];
        }
        
        return answer;
    }
}

File: LEVEL1/완주하지 못한 선수.cpp
#include<string>
#include<map>
#include<vector>
using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    map<string, int> mp;
    
    for(auto i : completion) {
        mp[i]++;
    }
    
    for(auto i : participant) {
        mp[i]--;
        
        if(mp[i] < 0) {
            return i;
        }
    }
}